### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by jamierocks** [SpongePowered/SpongeHome#11]
![image](https://cloud.githubusercontent.com/assets/5103549/14403100/bfdb2f26-fe42-11e5-911f-340295fed393.png)

I experimented around and the following should work.

``` scss
// iPhone 4/4s to iPhone 6
@media only screen
and (min-device-width: 320px)
and (max-device-width: 667px)
and (-webkit-min-device-pixel-ratio: 2) {
    .col-xs-2 {
        width: 100%;
    }
    .col-xs-offset-1 {
        margin-left: 0;
    }
}

// iPhone 6+
@media only screen
and (min-device-width: 414px)
and (max-device-width: 736px)
and (-webkit-min-device-pixel-ratio: 3) {
    .col-xs-2 {
        width: 100%;
    }
    .col-xs-offset-1 {
        margin-left: 0;
    }
}
```

With those changes in place you get: ![image](https://i.lexteam.xyz/Kk0gtpD7T8.png)

**Comment by MichaelHillcox** [comment]
I'm on it. I don't know why I am working on this now but meh. Might as well. Not nothing else on the books

**Comment by MichaelHillcox** [comment]
I've slightly improve upon this. Those Icons layouts are very odd :P

**Comment by progwml6** [comment]
what about android & tablets?

**Comment by jamierocks** [comment]
Oh that was example code - also although it specifically targets iPhones it works with most phones

**Comment by Tzky** [comment]
Is this still a concern? I pulled some changes in a few days ago which changed some parts of the page.

**Comment by MichaelHillcox** [comment]
@Tzky I've mostly fixed it but its impossible to test all devices with out actually having them.

**Comment by progwml6** [comment]
What did you check curiously as I'm sure we have someone around with other devices?

**Comment by jamierocks** [comment]
Use the mobile view of Chrome (or whatever browser you use) :p

**Comment by ZephireNZ** [comment]
I was told to post this here: http://i.imgur.com/Wg3V4V2.png

Nexus 5X running Marshmallow and Chrome 50

**Comment by ZephireNZ** [comment]
The little black box is caused by pressing the hamburger button, which seems to be out of place. I imagine the hovering buttons were meant to be inside there instead.

**Comment by Tzky** [comment]
@ZephireNZ yes, we're (or i am...) aware that the buttons and dropdown on the navbar are currently broken when viewing the page on mobile. I'll fix asap ;)

I broke mobile support when i added the new buttons.

**Comment by jamierocks** [comment]
Yeah that's a bug with all mobiles currently

**Comment by Tzky** [comment]
The mobile issues should be fixed as of 62f45b9 and 31644ba
